entries/z/zerotier.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "ZeroTier": {
+    "domain": "zerotier.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://accounts.zerotier.com/realms/zerotier/account/#/account-security/signing-in",
```

Comment:
The documentation cannot be behind a login screen. Please remove  the link and post a screenshot of the 2FA setup process or login process.

> [!WARNING]
> Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)
```suggestion
```